RELEASES.md

```diff
@@ -27,6 +27,12 @@
 - `bevy` dependency has been bumped from 0.12 to 0.13.
 - `bevy_egui` dependency has been bumped from 0.24 to 0.25.
 
+## Version 0.12.2

```

Comment:
This should probably be `0.13.1`. I don't think this is impactful enough to backport without others asking for it.